extensions/graphql/src/graphql.container.ts

```diff
@@ -12,13 +12,12 @@ import {
   filterByKey,
   filterByServiceInterface,
 } from '@loopback/core';
+import {MIDDLEWARE_CONTEXT} from '@loopback/rest';
 import {ExpressContext} from 'apollo-server-express/dist/ApolloServer';
 import debugFactory from 'debug';
 import {ContainerType, ResolverData} from 'type-graphql';
 import {GraphQLBindings, GraphQLTags} from './keys';
-
 const debug = debugFactory('loopback:graphql:container');
-const MIDDLEWARE_CONTEXT = Symbol.for('loopback.middleware.context');
```

Comment:
I think it's by design that we don't want to have hard dependency on `@loopback/rest`.

`Symbol.for('loopback.middleware.context')` resolves a global symbol by name if it exists. In our case, there are two possibilities:

1. There are different versions of `@loopback/express` modules in the same app dependency tree
2. The module loading order matters. (If graphql.container.js is loaded before packages/express/src/types.ts (line 330). Try to change this line to be `Symbol.for` as well may fix the issue.

Reply:
Okay, testing that out :)

Reply:
This works, I am pushing that change now :)